crates/environ/src/vmoffsets.rs

```diff
@@ -91,6 +91,10 @@ pub struct VMOffsets<P> {
     defined_globals: u32,
     defined_func_refs: u32,
     size: u32,
+
+    // NOTE(dhil): The following field is used as "global" to store
+    // the arguments of continuations and payloads of suspensions.
+    typed_continuations_store: u32,
```

Comment:
If the `typed_continuations_store` stays here in the longer term we should add it to the comment at the beginning of the file.